src/App.js

```diff
@@ -1,9 +1,13 @@
 import Header from './containers/Header/Header';
+import Footer from './containers/Footer/Footer';
+import Body from './containers/Body/Body';
+
+import './App.css';
 
 function App() {
   return (
     <div className="App">
-      <Header />
+      <div className="Header"><Header /></div>
```

Comment:
Why was classname header added?

Reply:
I was going to use it for the css page for the App.js, but realised its not needed as they already have their own style sheets.